[org] integrate org-appear with evil when triggering manually (#15284)
syl20bnr/spacemacs: layers/+emacs/org/README.org

   '((org :variables org-enable-appear-support t)))
#+END_SRC

> If you set =org-appear-trigger= to =manual= and your editing style is modal,

Reviewer: modal? do you mean vim or hybrid?

Author: Yes. Would it be better to phrase it that way?

Reviewer: yes, please change it.

Author: Will do when I get a sec.

Reviewer: ```suggestion
If you set =org-appear-trigger= to =manual= and your editing style is =vim= or =hybrid=,
```

> =org-appear= will be on in insert mode but off in normal mode.

Reviewer: ```suggestion
=org-appear= is turned on in insert mode but not in normal mode.
```
